packages/vats/test/bootstrapTests/test-vaults-integration.js

```diff
@@ -18,6 +18,19 @@ const test = anyTest;
 // presently all these tests use one collateral manager
 const collateralBrandKey = 'IbcATOM';
 
+const likePayouts = (collateral, minted) => ({
+  Collateral: {
+    value: {
+      digits: String(collateral * 1_000_000),
```

Comment:
I don't think this will survive the move to smallcaps (#6822).

Not critical. We'll notice when it's time.